torch/csrc/jit/script/init.cpp

```diff
@@ -363,6 +346,37 @@ Resolver pythonResolver(ResolutionCallback rcb) {
   };
 }
 
+FunctionSchema getSchemaWithDefaults(
+    std::unordered_map<std::string, py::object>& default_args,
+    FunctionSchema& schema) {
+  std::vector<Argument> new_args;
+  for (auto& arg : schema.arguments) {
+    auto it = default_args.find(arg.name);
+    if (it != default_args.end()) {
+      IValue value = *constantToIValue(it->second);
+      if (arg.type->isSubtypeOf(DynamicType::get()) && value.isScalar()) {
```

Comment:
Why is this needed? We don't convert things to tensors by default in other cases, and in Python these numbers will be scalars not tensors on entry to the function.